.github/workflows/reuse.yaml

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: 2023 The binaryornot Authors, see AUTHORS.rst
```

Comment:
IMHO the copyright should be `Copyright (c) 2013, Audrey Roy and others.`

Reply:
Is there a standard way to describe this? I'm fine with both (though I would like to reference the Authors file) but I'm wondering what's the best way.